From 83d2dbeca0f712b48c5da739f877df6094d242ce Mon Sep 17 00:00:00 2001
From: mushtaqk07 <Mushtaq.Khan@amd.com>
Date: Thu, 2 Feb 2023 11:35:42 +0000
Subject: [PATCH 06/11] Add Pinctrl for EMMC 8-bit configs

This patches add pin control configuration for EMMC 8-bit mode.
---
 arch/arm/boot/dts/aspeed-g6-pinctrl.dtsi | 5 +++++
 1 file changed, 5 insertions(+)

diff --git a/arch/arm/boot/dts/aspeed-g6-pinctrl.dtsi b/arch/arm/boot/dts/aspeed-g6-pinctrl.dtsi
index 7cd4f075e325..659bdfecbad5 100644
--- a/arch/arm/boot/dts/aspeed-g6-pinctrl.dtsi
+++ b/arch/arm/boot/dts/aspeed-g6-pinctrl.dtsi
@@ -857,6 +857,11 @@
 		groups = "EMMCG4";
 	};
 
+        pinctrl_emmcg8_default: emmcg8_default {
+                function = "EMMC";
+                groups = "EMMCG8";
+        };
+
 	pinctrl_sgpm1_default: sgpm1_default {
 		function = "SGPM1";
 		groups = "SGPM1";
-- 
2.17.1

